cPP_DSC.adoc

```diff
@@ -903,43 +936,43 @@ FCS_COP.1.1/SigGen:: The TSF shall perform _digital signature generation_ in acc
 |List of Standards
 
 |RSA-PKCS
-|RSASSA-PKCS1-v1_5 using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#] 
+|RSASSA-PKCS1-v1_5 using {empty}[*selection*: [.underline]#SHA-224, SHA-256, SHA-384, SHA-512, SHA3-224, SHA3-256, SHA3-384, SHA3-512#]
 |{empty}[*selection*: [.underline]#2048, 3072, 4096#] bits 
 |{empty}[*selection*: [.underline]#RFC 8017, PKCS #1 v2.2 (Section 8.2); FIPS PUB 186-5 (Section 5.4)#] [RSASSA-PKCS1-v1_5]
 
 {empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 180-4, FIPS PUB 202#] [SHA]
 
 |RSA-PSS
-|RSASSA-PSS using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#]
+|RSASSA-PSS using {empty}[*selection*: [.underline]#SHA-224, SHA-256, SHA-384, SHA-512, SHA3-224 SHA3-256, SHA3-384, SHA3-512#]
 |{empty}[*selection*: [.underline]#2048, 3072, 4096#] bits
 |{empty}[*selection*: [.underline]#RFC 8017, PKCS#1v2.2 (Section 8.1); FIPS PUB 186-5 (Section 5.4)]# [RSASSA-PSS]
 
 {empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 180-4, FIPS PUB 202#] [SHA]
 
 |ECDSA
-|ECDSA on {empty}[*selection*: [.underline]#brainpoolP256r1, brainpoolP384r1, brainpoolP512r1, NIST P-256, NIST P-384, NIST P-521#] using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#]
-|{empty}[*selection*: [.underline]#256, 384, 512, 521#] bits
```

Comment:
It would be more consistent with FCS_CKM_EX.7 to leave out the "521" here for P-521. It's technically not accurate to leave out "521", but probably less confusing.